### DISCUSSION TITLE
possible to add Fixed Column ?

### DISCUSSION THREAD

**Discussion by ridwanmubarok** [rappasoft/laravel-livewire-tables#813] [Feature Requests]
thanks

**Comment by PieterCappelle** [comment]
Any update on this, or anyone found a solution?

**Reply by lrljoe** [reply]
There's no context available, so I don't know what is meant by "Fixed Column"

**Comment by PieterCappelle** [comment]
The meaning of this feature request is that the last column of a table should be fixed, and not scrollable inside the container. As example, you have a table with 20 columns, and the last column is with buttons like "Edit" or "Delete". We receive a lot of feedback from customers that don't see those call to actions. Everytime we need to tell them to scroll "horizontal" to see the last column. We want to set the last column fixed and always visible in the viewport. You understand?

**Reply by lrljoe** [reply]
So "column 20" should always be visible, regardless of horizontal scroll.

This is possible to do, but not smooth

**Reply by PieterCappelle** [reply]
Yes, exactly, just so we know, are you trying to implement this?
